### DISCUSSION TITLE
HA Entity IDs

### DISCUSSION THREAD

**Discussion by RunningTim** [Arnold-n/P1P2MQTT#107] [Q&A] [answered]
Hi @Arnold-n,

Thanks for the great project - I recieved my two P1P2 units here in Spain within a week and the initial setup of the first unit has proved to be really straightforward with my F-series AirCon.  However I have a query regarding the created Home Assistant entity ids:

You kindly renamed one of my units bridge1 and I can see that this is used in the MQTT discovery data ( eg. Topic: homeassistant/sensor/bridge1/P1P2MQTT_bridge1_T1_Setpoint_Temperature_1_1/config ).  However, neither the individual sensor entities nor the Climate domain created in HA include the "bridge1" tag ( eg. sensor.hc_setpoints_setpoint_temperature_1  climate.hc_setpoints_setpoint_cooling ).  Is this expected behaviour?

I could rename all the discovered MQTT devices in HA to distinguish between the two units, but presumably, this would be a problem if the MQTT discovery process was to re-run.

**Comment by Arnold-n** [comment, marked as answer]
Hi @RunningTim , thanks for your feedback, and oops, sorry, my mistake, I should have made two more configuration changes for you. Parameters 26 (use bridge name for HA device name) and 27 (use bridge name in device identifier) should be 1 in both bridges. You can do so by sending the following payload messages on `P1P2/W` or using telnet:
`P26 1`
`P27 1`
`D5` to save
`D0` to restart
wait ~20 seconds
`D14` to remove old retained MQTT messages
wait until ready, may take 1 or 2 minutes
`D3` to rebuild new configuration messages

This should give you separate bridge0_* and bridge1_* subdevices instead of the usual HC_* subdevices.

Please let me know if this doesn't work.

**Reply by RunningTim** [reply]
Thanks @Arnold-n

I have had a play with the configuration settings:  Renamed both the device and bridge parameters and also enabled P22.  I now have my ideal device and individual entity IDs ( eg.  HVAC_Upper_Sensors and hvac_upper_sensors_temperature_room ).  I will apply a similar setup to the HVAC_Lower unit.

All sorted :)
